coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -397,13 +406,10 @@ function AclApprovals() {
             noWrap={false}
           />
         }
-        pagination={
-          <Pagination
-            activePage={data.currentPage}
-            totalPages={data.totalPages}
-            setActivePage={handleChangePage}
-          />
-        }
+        pagination={pagination}
+        isLoading={isLoading}
+        isErrorLoading={isError}
```

Comment:
Do we need to pass both `isError` and `errorMessage`? Could we only pass `errorMessage`, and then if it's not `undefined`, render the error state?

Reply:
We don't always get an error message at this point. We _should_ since we have our additional check for the hidden errors, but I would not rely on that 100%. In that case, we would get an `isError` true from the query, but en empty error message. So it's an extra safety measure here.